SECURITY-RELEASE-PROCESS.md

```diff
@@ -0,0 +1,195 @@
+# Security Release Process
+
+CoreDNS project has adopted this security disclosures and response policy
+to ensure a responsible handle of critical issues.
+
+
+## Product Security Team (PST)
+
+Security vulnerabilities should be handled quickly and sometimes privately. 
+The primary goal of this process is to reduce the total time users are vulnerable to publicly known exploits.
+
+The Product Security Team (PST) is responsible for organizing the entire response including internal communication and external disclosure.
+
+The initial Product Security Team will consist of set of maintainers that volunteered.
+The list is available in this [document](/OWNERS.md) and is synchronized with the receivers of product-security-team@coredns.io list.
+
+## Disclosures
+
+### Private Disclosure Processes
+
+If you find a security vulnerability or any security related issues, 
+please DO NOT file a public issue - that means do not create a Github issue, 
+instead send your report privately to security@coredns.io. 
+Security reports are greatly appreciated and we will publicly thank you for it.
+
+if you know it, please provide as much information as possible, so we can react quickly.
+For instance, that could include:
+- Description of the location and potential impact of the vulnerability;
+- A detailed description of the steps required to reproduce the vulnerability (POC scripts, screenshots, and compressed screen captures are all helpful to us)
```

Comment:
I think you mean "compressed packet captures"?

Reply:
yes.